client/package.json

```diff
@@ -20,8 +20,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-router": "^6.4.2",
-    "react-router-dom": "^6.4.2",
-
+    "react-router-dom": "^6.4.2"
```

Comment:
removed unnecessary comma (this change was made in previous commit webpack config changes but hasnt been merged yet)